### PR TITLE
Initial pass at expanding the README for this project

### PR DESCRIPTION
**Pull request by rhempel**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
...
+---------
+
+`Getting Started with ev3dev <ev3dev-getting-started_>`_
+    If you got here as the result of looking for "how to program
+    LEGO MINDSTORMS EV3 using Python" then you might not be aware that
+    this is part of a much larger project called ev3dev_. Make sure
+    you read the `Getting Started <ev3dev-getting-started_>`_ page
+    to become familiar with ev3dev_ first!
+
+`Connecting the EV3 to the Internet via USB <ev3dev-usb-internet_>`_
```
Comment (WasabiFan) [inline README.rst]: I wouldn't recommend directing them to the USB tutorial. Maybe tell them to look at all the options listed on the tutorials page instead? Actually, we might need a networking page on the site...

File: README.rst [hunk 2]
```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
...
+then use ``easy-install`` to do the rest.
+
+Once you have your EV3 `connected to the Internet <ev3dev-usb-internet_>`_
+you must update the package lists and install the ``python3-pil`` package::
+
+  sudo apt-get update
+  sudo apt-get install python3-pil
+
+Please be patient - the EV3 will be *much* slower than you are used to, but rest
+assured it it going as fast as it can. A typical ``apg-get update`` will take about
```
Comment (WasabiFan) [inline README.rst]: `apg` -> `apt`

File: README.rst [hunk 3]
```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
...
+    for Linux, Windows, and Mac computers.
+
+`Demo Robot <demo-robot_>`_
+    Laurens Valk of robot-square_ has been kind enough to allow us to
+    reference his excellent EXPLOR3R robot. Consider building the
+    EXPLOR3R and running the demo programs referenced below to get
+    familiar with what Python programs using this binding look like.
+
+`Demo Code <demo-code_>`_
+    There are demo programs that you can run to get acquanited with
```
Comment (ddemidov) [inline README.rst]: typo: acquanited

File: README.rst [hunk 4]
```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
     :target: https://travis-ci.org/rhempel/ev3dev-lang-python
 .. image:: https://readthedocs.org/projects/python-ev3dev/badge/?version=latest
     :target: http://python-ev3dev.readthedocs.org/en/latest/?badge=latest
-    :alt: Documentation Statu
+    :alt: Documentation Status
 
-This is a python library implementing unified interface for ev3dev_ devices.
+A Python library implementing unified interface for ev3dev_ devices.
```
Comment (ddemidov) [inline README.rst]: How about adding a couple of examples here? Something like

Usage
--------

Run large motor connected to output port A for 3 seconds:
```py
>>> import ev3dev.ev3 as ev3
>>> m = ev3.LargeMotor('outA')
>>> m.connected
True
>>> m.run_timed(time_sp=3000, duty_cycle_sp=75)
```

Make your robot speak:
```py
>>> ev3.Sound.speak('Welcome to the EV3 project!').wait()
```

Use left led to indicate touch sensor state:
```py
>>> ts = ev3.TouchSensor()
>>> while True:
...    ev3.Leds.set_color(ev3.Leds.LEFT, ts.value() and ev3.Leds.RED or ev3.Leds.GREEN) 
```

File: README.rst [hunk 5]
```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
...
 
-.. _ev3dev: http://ev3dev.org
+Resources
+---------
+
+`Getting Started with ev3dev <ev3dev-getting-started_>`_
```
Comment (ddemidov) [inline README.rst]: There is something wrong with the links in the headings. All of them result in 404 error.

Comment (ddemidov) [inline README.rst]: Same applies to most of the links inside the text.

File: README.rst [hunk 6]
```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
...
+    If you got here as the result of looking for "how to program
+    LEGO MINDSTORMS EV3 using Python" then you might not be aware that
+    this is part of a much larger project called ev3dev_. Make sure
+    you read the `Getting Started <ev3dev-getting-started_>`_ page
```
Comment (ddemidov) [inline README.rst]: This should be
```
`Getting Started`_
```
with the corresponding
```
_Getting started: http://www.ev3dev.org/docs/getting-started/
```
in the end.

File: README.rst (hunk 4 above)
Comment (rhempel) [inline README.rst]: Thks, I have reorganized this a bit to make it easier for users to simply cut/paste the code (hiding the prompt).

File: README.rst (hunk 5 above)
Comment (rhempel) [inline README.rst]: The intent was to use the RST ability to have a single logical link but different link text - it works on my local machine - but clearly the GitHub rst processor has some trouble with this style of link...I'll investigate

File: README.rst (hunk 1 above)
Comment (rhempel) [inline README.rst]: What I'll do is point them at all of the networking options - USB is the minimal method if they don't have a compatible dongle and can't get BT to work. They must have some method of getting to a shell, and USB is the easiest one (IMHO)

File: README.rst (hunk 5 above)
Comment (rhempel) [inline README.rst]: This is a defect in GitHub's handling of this style of linking:

```
`Getting Started with ev3dev <ev3dev-getting-started_>`_
...
.. _ev3dev-getting-started: http://www.ev3dev.org/docs/getting-started/
```
I'll open a defect with GitHub

File: README.rst (hunk 1 above)
Comment (WasabiFan) [inline README.rst]: > USB is the easiest one (IMHO)

I am a bit biased as I spent several months fighting with USB when I first started with ev3dev.

**Pull request merged**